src/display/annotation_layer.js

```diff
@@ -181,7 +180,7 @@ class AnnotationElement {
       this.container = this._createContainer(ignoreBorder);
     }
     if (createQuadrilaterals) {
-      this.quadrilaterals = this._createQuadrilaterals(ignoreBorder);
+      this.quadrilaterals = this._createQuadrilaterals();
```

Comment:
Given that `_createQuadrilaterals` now always return `undefined`, this assignment looks slightly strange now.

However it's not clear to me Popup-annotations still work correctly, given that the following code now *by definition* won't ever run:
https://github.com/mozilla/pdf.js/blob/52796615fb4470d9f16e568971696286b6c6b354/src/display/annotation_layer.js#L468-L471